FluentUI.Demo/src/main/res/values/strings.xml

```diff
@@ -264,6 +264,17 @@
     <string name="list_item_click_custom_accessory_view">You clicked on the custom accessory view.</string>
     <string name="list_item_click_sub_header_custom_accessory_view">You clicked on the sub header custom accessory view.</string>
 
+    <!--ListItem V-Next-->
```

Comment:
Localization of these strings is not required?

Reply:
In Demo app, we do not have localization currently for any of the string